src/datalad_installer.py

```diff
@@ -2155,7 +2155,7 @@ def parse_header_links(links_header: str) -> Dict[str, Dict[str, str]]:
 
         {"next": {"url": "...", "rel": "next"}, "last": { ... }}
     """
-    # <https://git.io/JcYZi>
+    # <https://github.com/psf/requests/blob/c45a4df/requests/utils.py#L829>
```

Comment:
note: that blob implementation is slightly different (returns list not dict) so either this is improved version from some later blob or adjusted in our copy.